package.json

```diff
@@ -17,10 +17,10 @@
     "build:icons": "yarn react build:icons",
     "test": "yarn react test && yarn e2e test",
     "prepare": "husky install",
-    "version:next": "yarn changeset version --snapshot next-$(git rev-parse --short=7 HEAD) && yarn build",
+    "version:next": "yarn changeset version --snapshot next-$(git rev-parse --short=7 HEAD) && yarn angular build",
```

Comment:
This improves build times because everything should already be built, `changeset version` updates versions, but `yarn angular build` _rebuilds_ with the correct version?

Reply:
See my comment. below, it's actually unneeded. But yes, that was my original intention because other packages were already built from `yarn --frozen-lock`.